docs/phases_and_modules.md

```diff
@@ -45,23 +45,27 @@ way with the participation module's content.
     -   weight
 -   phases are ordered by weight
     -   weights are usually initialised from the position of the phase type in
-        the blueprint
+        the blueprint.
+        -   Attention: _deprecated_ as this only works for one module where the
+            phase order is determined by weight.
 -   no checking of start and end dates between phases (if overlap)
--   view of phase displayed from start date to start date of next phase / or indefinitely
+-   view of phase displayed from start date to start date of next phase / or
+    indefinitely?
+-   view of phase displayed from start date to end date
 -   permissions are only set for participants
-    -   moderators and initiators can always do everything
+    -   ~~moderators and~~ initiators can always do everything?
```

Comment:
Yes, on the item level this is right. Only on project level moderators cannot do anything